ReactiveCocoaTests/UIKit/UIKeyboardSpec.swift

```diff
@@ -0,0 +1,37 @@
+import UIKit
+import ReactiveSwift
+import ReactiveCocoa
+import Quick
+import Nimble
+
+class UIKeyboardSpec: QuickSpec {
+	override func spec() {
+		describe("UIView.Type.reactive.keyboardChange") {
+			it("should emit a `value` event when the notification is posted") {
+				var context: KeyboardChangeContext?
+
```

Comment:
No need to give the parameter a name if it's going to be abbreviated :P just use `$0`?